client/src/templates/Challenges/fill-in-the-blank/show.tsx

```diff
@@ -0,0 +1,456 @@
+    const { answersCorrect } = this.state;
+
+    if (answersCorrect[index] === true) {
+      return 'green-underline';
+    } else if (answersCorrect[index] === false) {
+      return 'red-underline';
+    } else {
+      return '';
+    }
+  }
+
+  render() {
+    const {
+      data: {
+        challengeNode: {
+          challenge: {
+            title,
+            description,
+            instructions,
+            superBlock,
+            block,
+            translationPending,
+            fields: { blockName },
+            fillInTheBlank: { sentence, blanks },
+            audioPath
+          }
+        }
+      },
+      openHelpModal,
+      pageContext: {
+        challengeMeta: { nextChallengePath, prevChallengePath }
+      },
+      t,
+      isChallengeCompleted
+    } = this.props;
+
+    const blockNameTitle = `${t(
+      `intro:${superBlock}.blocks.${block}.title`
+    )} - ${title}`;
+
+    const { allBlanksFilled, feedback, showFeedback, showWrong } = this.state;
+
+    const splitSentence = sentence.replace(/^<p>|<\/p>$/g, '').split('_');
+    const blankAnswers = blanks.map(b => b.answer);
+
+    return (
+      <Hotkeys
+        executeChallenge={() => this.handleSubmit()}
+        containerRef={this.container}
+        nextChallengePath={nextChallengePath}
+        prevChallengePath={prevChallengePath}
+      >
+        <LearnLayout>
+          <Helmet
+            title={`${blockNameTitle} | ${t('learn.learn')} | freeCodeCamp.org`}
+          />
+          <Container>
+            <Row>
+              <Spacer size='medium' />
+              <ChallengeTitle
+                isCompleted={isChallengeCompleted}
+                translationPending={translationPending}
+              >
+                {title}
+              </ChallengeTitle>
+
+              <Col md={8} mdOffset={2} sm={10} smOffset={1} xs={12}>
+                <PrismFormatted text={description} />
+                {audioPath && (
+                  <>
+                    {' '}
+                    <Spacer size='small' />
+                    <Spacer size='small' />
+                    {/* TODO: Add tracks for audio elements */}
+                    {/* eslint-disable-next-line jsx-a11y/media-has-caption*/}
+                    <audio className='audio' controls>
+                      <source
+                        src={`https://cdn.freecodecamp.org/${audioPath}`}
+                        type='audio/mp3'
+                      ></source>
+                    </audio>
+                  </>
+                )}
+                <Spacer size='medium' />
+                <PrismFormatted text={instructions} />
+                <Spacer size='medium' />
+                <h2>{t('fill-in-the-blank')}</h2>
+                <Spacer size='small' />
+                <ObserveKeys>
+                  <div>
+                    <p>
+                      {splitSentence.map((s, i) => {
+                        return (
+                          <>
```

Comment:
```suggestion
                          <Fragment key={i}>
```
Just to suppress the warning (also, we know that `i` doesn't change between renders)

Reply:
fixed in latest commit